ui/app/models/job.js

```diff
@@ -327,4 +328,16 @@ export default class Job extends Model {
   // spec first. In order to preserve both the original HCL and the parsed response
   // that will be submitted to the create job endpoint, another prop is necessary.
   @attr('string') _newDefinitionJSON;
+
+  @computed('variables', 'parent', 'plainId')
+  get pathLinkedVariable() {
+    if (this.parent.get('id')) {
```

Comment:
`thought` `suggestion`:  The `get` method is a little dangerous because we don't know if the request is synchronous or asynchronous. A work around that we use for getting the `id` is using `this.belongsTo('$ModelName').id()`. And a slightly hackier workaround to escape the behavior of a promise is using `model.content`.